include/seqan3/io/alignment_file/output.hpp

```diff
@@ -727,22 +787,39 @@ class alignment_file_output
  * \relates seqan3::alignment_file_output
  * \{
  */
+
+/*!\brief Deduces selected_field_ids from input and sets alignment_file_output::ref_ids_type to
+ * seqan3::detail::ref_info_not_given. Valid formats and stream_char_type are defaulted. */
```

Comment:
In our code I usually see the block comment closers on the next line:
```cpp
/*!
 *
 */
```
If this is part of SeqAn's code style, you may want to change this (+ 2x below).

Reply:
Also normally we don't have two sentences in the brief. Looks like someone is trying to avoid adding a full documentation entry 😆

Reply:
Type deduction guides only ever got a brief :disappointed_relieved: The names are just too long